### ISSUE TITLE
Предлагаю памятку

### ISSUE THREAD

**Issue by meganafig** [freedomsex/freedomsex#489]
Дамы и Господа,все мы видим периодически хамство и неуважение в открытом виде.почему бы не забомбить новоприбывшим маленькую(короткую,что бы прочли)нотацию об уважении выбора.кому то не нравится минет))образно говоря..может пусть научатся ставить себя на место другого?типа не суди...

**Comment by meganafig** [comment]
Видимо тупое предложение,простите))

**Comment by tebaly** [comment]
Сумбурное. Попробуйте ещё раз :stuck_out_tongue:

**Comment by inquisitor6789** [comment]
да не, нормальное предложение. Только все класть на него болт будут. Безнаказанность интернета. И ничего тут не поделаешь. Просто все эти чудики в реале такое никогда не предложат ибо могут сильно по щам схватить. Вот в сети и отрываются,

**Comment by tebaly** [comment]
Да. Кстати. Уже с давних при давних времен на сайте практикуется _атака оскорблениями_. Здесь нужно пояснить. Хамов и беспередельшиков хватает всегда, но здесь наши друзья по цеху, используют всё до чего могут додуматься. И вот как-то уже давно придумали... отправлять девушкам гадости в автоматическом режиме. Тобишь ботами. А для обычных людей
* Есть правла, там всё расписано. 
* Была "нотация", из-за смены дизайна она пропала.
* Надо её как-то вернуть... да, спасибо :)

До ботов скоро дойдем, об этом уже был анонс :stuck_out_tongue:
